### DISCUSSION TITLE
MEO SERVERS storage code CHANNELS differently??

### DISCUSSION THREAD

**Discussion by spotanjo3** [LITUATUI/M3UPT#70] [General]
Hello. And good afternoon/good evening.

I noticed there are different codes about MEO channels. For an example: SPORT TV 5

My meo streaming m3u8 showing that SPORT TV 5 and SPORT TV 4 are - LIVE$SPORTTV_GOLFE_HD (SPORT TV 5) and LIVE$SPORTTVLive_HD (SPORT TV 4)

Do you have an accurate meo streaming m3u8 code somewhere ? I cannot find one.

If I rename it to LIVE$SPORTTV5_HD and SPORTTV4_HD and it wont worked. So I have to leave them untouched follwing to MEO Server code unknown and its working ok. I dont mind because I liked them the way they are. So I knew that MEO server read it as SPORTTV_GOLFE_HD and SPORTTVLive_HD only as well as other channels also.

There are many channels I have are storage by MEO server different from yours. I prefer to leave it untouched. Yeah, EPG can read them fine through TIVIMate select by select each instead of rename them.

Anyway, I want to know what MEO SERVER storage code lists like this ? I am curious about it. Do you know? Sorry about my english.

**Comment by LITUATUI** [comment]
Check [here](https://github.com/LITUATUI/M3UPT/blob/main/EPG/meo.pt.channels.xml) for MEO channel names.
